src/main/java/seedu/address/model/person/StarsLessThanPredicate.java

```diff
@@ -0,0 +1,43 @@
+package seedu.address.model.person;
+
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.ToStringBuilder;
+
+/**
+ * Tests that a {@code Person}'s {@code Name} matches any of the keywords given.
+ */
+public class StarsLessThanPredicate implements Predicate<Person> {
+    private final Integer upperBound;
+
+    public StarsLessThanPredicate(Integer upperBound) {
```

Comment:
You can check if `upperBound` is less than 1, if it is 0, then it will not output any students since no students should have negative stars.

Reply:
I have updated my code to enforce this in the command parser.